src/views/Support/Support.js

```diff
@@ -132,6 +132,29 @@ const useStyles = makeStyles((theme) => ({
 
 let socket = null;
 
+const pageSize = 20;
+
+const buildUserQuery = (userPage, userOffset) => {
+  let query = {};
+  let s_query = {
+    where: {},
+    options: {}
+  };
+  
+  s_query.options.limit = pageSize;
+  s_query.options.skip = userOffset + pageSize * userPage;
+
+  query.query = JSON.stringify(s_query);
+
+  return query;
+};
+
+
+const queryUser = async (uPage, uOffset) => {
```

Comment:
1. should create ApiCustomerService.js
2. queryUser should be in ApiCustomerService.js